### DISCUSSION TITLE
Ability to drag&drop legend values in Time Series panel

### DISCUSSION THREAD

**Discussion by szerwi** [grafana/grafana#37667] [Dashboards, Panels & Visualizations]
**What would you like to be added:**
Possibility to drag&drop legend values in Time Series panel.

**Why is this needed (describe your use case and goals):**
It will be much easier to change the order of legend values. Currently, if you need to change the order, you have to delete all the values and add them again in the correct order.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
